openrtb.go

```diff
@@ -221,8 +221,8 @@ type Producer ThirdParty
 // (such as IP geo lookup), or by user registration information (for example provided to a publisher
 // through a user registration).
 type Geo struct {
-	Lat           float64   `json:"lat,omitempty"`           // Latitude from -90 to 90
-	Lon           float64   `json:"lon,omitempty"`           // Longitude from -180 to 180
+	Lat           *float64  `json:"lat,omitempty"`           // Latitude from -90 to 90; *float64 allows checking for nil, float64 would default to 0, which is a valid latitude
```

Comment:
The assumption is/was that lat/lon are never going to be exactly 0 (as this is in the middle of the sea), hence validation is pretty straight-forward: `if geo.Lat != 0 || geo.Lon != 0 { // do something with lat/lon`

Reply:
You're right, if lat, lon == (0, 0) it's probably fake data too. I'll close the PR.